app/controllers/requests_controller.rb

```diff
@@ -17,7 +17,7 @@ def new
   end
 
   def current_request
```

Comment:
@cbeer no, I have just gotten in a habit of doing that in the case that I don't have an implementation in mind so that nobody blindly uses the method w/o knowing what the intention is.  Feel free to do something else w/ that method now that you have a necessary implementation.